src/drawtypes/label.cpp

```diff
@@ -218,13 +218,21 @@ namespace drawtypes {
     }
 
     size_t maxlen = conf.get(section, name + "-maxlen", 0_z);
-    bool ellipsis = conf.get(section, name + "-ellipsis", true);
+    bool has_ellipsis = conf.has(section, name + "-ellipsis");
+    bool ellipsis = conf.deprecated_type_changed(section, name + "-ellipsis", name + "-ellipsis-string", true);
+    string ellipsis_str = conf.get(section, name + "-ellipsis-string", "..."s);
 
-    if(ellipsis && maxlen > 0 && maxlen < 3) {
+    if (has_ellipsis && ellipsis && ellipsis_str.empty()) {
+      throw application_error(
```

Comment:
If `-ellipsis-string` is set, `-ellipsis` should probably be ignored.

Reply:
Fixed by https://github.com/polybar/polybar/pull/1673/commits/0b005696146b6f5d722cf7c7527ef5b3aa65a038